yb-voyager/cmd/common.go

```diff
@@ -107,7 +107,8 @@ func updateFilePaths(source *srcdb.Source, exportDir string, tablesProgressMetad
 			targetTableName := tname
 			// required if PREFIX_PARTITION is set in ora2pg.conf file
 			if tablesProgressMetadata[key].IsPartition {
-				targetTableName = tablesProgressMetadata[key].ParentTable + "_" + targetTableName
+				_, parentTable := tablesProgressMetadata[key].ParentTable.ForCatalogQuery()
```

Comment:
Do we need to add the parent table in the datafile paths?

Reply:
For oracle this might not be kicking as the IsPartition is always false.
I think if it's unnecessary then we can skip this change we can do it when required.

Reply:
Ah, okay now understood this else case for ora2pg - where it is not required to handle anything partition specific naming. So we can safely remove this if condition.